scripts/convert_ipynb_to_py.sh

```diff
@@ -1,21 +1,34 @@
 #!/bin/bash
 
+# show commands being executed (for debugging purposes)
+#set -x
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
 NOTEBOOK="${1}"
 if [ -z "${NOTEBOOK}" ]; then
   echo "Provide the notebook path"
   exit 1
 fi
 
+# capture whether notebook has a python or R kernel
+regex="\"file_extension\": \"(\.[a-zA-Z]+)\"\,"
```

Comment:
You may want to add the `$` end of line character to your regex to ensure that the pattern that looks like an extension is actually coming at the end of the file path

Reply:
Yes, I tried that. But in this case, the part in json file (.ipynb) where `file_extension` is located looks like this:
![image](https://user-images.githubusercontent.com/172687/150557578-5835b4e7-34b4-4524-a281-e8f821c58a33.png)
then I wondered what happens if instead of a newline, the json is saved with the next block following `file_extension`, which is valid. Adding a `$`, I think, would break that.